VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,245 @@
+* **Transaction fees:** The Hub maintains a whitelist of token that are accepted as fee payment. These fees are part of the total reward received by validators and delegators.
+
+These rewards are divided among validator’s staking pools according to each validator’s weight. Then, they are divided among delegators in proportion to each delegator’s stake.
+
+### What is the incentive to run a validator ?
+
+Validators earn additional rewards in the form of commissions on the fees that are collected. These fees can be collected not only in ATOMs but in any token that has been whitelisted by ATOM holders. 
+
+Validators also play a major role in governance. If a delegator does not vote, it inherits the vote from its validator. This gives validators a major responsibility in the ecosystem.
+
+### What is the probability of being chosen as a proposer for any given block? Does it go up with the quantity of ATOMs staked ?
+
+The validator that is selected to propose the next block is called proposer. The probability of being chosen is equal to the relative total stake (where total stake = self-bonded stake + delegators stake) of the validator. For example, if the total bonded stake across all validators is 100 ATOMs and a validator's total stake is 10 ATOMs, then this validator has a 10% probability to be chosen as the next proposer.
+
+### What is a commission?
+
+Fee rewards received by a validator’s pool are split between the validator and its delegators. The part that goes to the validator is called the validator’s commission. Each validator is free to set its initial commission, but there will be rules on how to change it.
+
+Note that validators cannot set commission on block rewards (also called inflationary rewards or provisions). This is to prevent a continuous shift in distribution from delegators to validators which would further the concentration of ATOMs and ultimately weaken the security of the system.
+
+### How are block rewards distributed?
+
+Block rewards are socially distributed, meaning that each block reward is distributed to each validator according to their stake.
+
+Let’s take an example where we have 10 validators with equal staking power and commission rate of 1%. Let’s also assume that the reward for a block is 1000 ATOMs and that each validator has 20% of self-bonded ATOMs. These tokens do not go directly to the proposer. Instead, they are evenly spread among validators. So now each validator’s pool has 100 ATOMs. These 100 ATOMs will be distributed according to each participant’s stake:
+* Validator will get 20 ATOMs
+* All delegators will get 80 ATOMs
+
+Then, each delegator can claim its part of the 80 ATOMs in proportion of their stake in the validator’s staking pool.
+
+### How are fees distributed?
+
+Fees are also socially distributed, but the block proposer can get a bonus on the fees of the block it proposes if it includes more than the strict minimum of required precommits.
```

Comment:
Fees are similarly distributed with the exception that the block proposer ...

Reply:
addressed